public/js/applicantForm.js

```diff
@@ -15,7 +15,14 @@
     checkedCohortDates();
 
     if (document.querySelector('#chosenCourseId')) {
-        outputCohorts(data.cohorts, document.querySelector('#chosenCourseId'));
+        const chosenCourseId = document.querySelector('#chosenCourseId').dataset.selected;
+        const futureCohorts = filterFutureDates(data.cohorts);
```

Comment:
this is doing a filter, could this be combined with the filter below to save looping the data twice?